include/albatross/src/models/sparse_gp.hpp

```diff
@@ -349,15 +382,14 @@ class SparseGaussianProcessRegression
   // which corresponds to the inverse square root of Sigma
   //
   //   Sigma = (B^T B)^-1
-  //
-  Eigen::ColPivHouseholderQR<Eigen::MatrixXd>
+  std::unique_ptr<Eigen::SPQR<Eigen::SparseMatrix<double>>>
   compute_sigma_qr(const Eigen::SerializableLDLT &K_uu_ldlt,
                    const BlockDiagonalLDLT &A_ldlt,
                    const Eigen::MatrixXd &K_fu) const {
     Eigen::MatrixXd B(A_ldlt.rows() + K_uu_ldlt.rows(), K_uu_ldlt.rows());
     B.topRows(A_ldlt.rows()) = A_ldlt.sqrt_solve(K_fu);
     B.bottomRows(K_uu_ldlt.rows()) = K_uu_ldlt.sqrt_transpose();
-    return B.colPivHouseholderQr();
+    return SPQR_create(B.sparseView(), Base::threads_.get());
```

Comment:
So we're keeping the QR of sigma sparse for both implementations? or should this be using the `QRImplementation`?

Reply:
Oops, I updated the update but not the initial fit!